src/Protocol/PortableContact.php

```diff
@@ -1471,7 +1471,9 @@ private static function discoverRelay($server_url)
 			$tags = [];
 			foreach ($data['tags'] as $tag) {
 				$tag = mb_strtolower($tag);
-				$tags[$tag] = $tag;
+				if (count($tag) < 100) {
```

Comment:
Please use `mb_strlen()` next time instead.

Reply:
Yeah, you are right.